src/loaders/express/index.ts

```diff
@@ -138,6 +139,7 @@ const loadExpressApp = async (connection: Connection) => {
   app.use('/transaction', VfnRouter)
   app.use('/billing', BillingRouter)
   app.use('/analytics', AnalyticsRouter)
+  app.use('/examples', ExamplesRouter)
```

Comment:
What do you mean by this?  Isn't the auth module already there? Those are auth checks, similar to the old authentication checks. We should do auth checks regardless, since some routes are protected

Reply:
I'm suggesting:

`app.use('/examples', authMiddleware, ExamplesRouter)` instead of the auth code being run in the individual examples controller functions

Reply:
it was a design decision to not have any middlewares, no?

Reply:
check with YR? I think he said that auth was okay as a middleware?

Reply:
cos if we move to a nested routes format, the tests will need to change later. very leceh

Reply:
@liangyuanruo

Reply:
yup, adding authentication is a good fit for the middleware pattern since there is a clear separation of functionality and there aren't any dependencies downstream. I was contemplating introducing decorators, but this way is probably more Express-like.

Reply:
Ok, I'll refactor auth as a middleware then, good for when we use tsoa when we update our backend endpoints to start with `/api`

Reply:
Will extract out authentication middleware as a separate PR.